src/repr/src/row/encoding.rs

```diff
@@ -167,13 +167,13 @@ pub trait DatumToPersist {
 
     /// Decodes the data in the persist column at the specific offset into a
```

Comment:
```suggestion
    /// Decodes the data with the given reference into a
```

(Or whatever you like!)

Reply:
whoops, done!